src/Windows/MainWindow.cs

```diff
@@ -626,5 +630,205 @@ private void exportAllResourcesToolStripMenuItem_Click(object sender, EventArgs
                 }
             }
         }
+
+       
+        /// <summary>
+        /// Fill resx with translations from google
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void translateToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            LaunchAutoTranslate("es");

```

Comment:
Hard-coded languages have limited usability for other people. It would be better to show a window with selection of column to be translated, and optionally the language of the column (so for example you can translate the `fr` column into other languages, even if it holds something other than French text. Or, more importantly, specify language for the `NoLanguageCode` column).

It would also be ideal to have a list of columns to google translate, so for example you can only translate `fr` column into the `de` column.

The google translate code should be extracted into a separate class, away from MainWindow as much as possible, so that in the future it is easier to add other translators.

Reply:
OK, I added Configuration > Change language for default resx which takes culture to translate from

Reply:
All code for translation that i can extract is in GTranslationServices, other methods uses MainWindow controls